eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -0,0 +1,105 @@
+parameters:
+  - name: PackageName
+    type: string
+    default: ""
+  - name: ServiceDirectory
+    type: string
+    default: ""
+  - name: EnvVars
+    type: object
+    default: {}
+  - name: MaxParallel
+    type: number
+    default: 0
+  - name: TimeoutInMinutes
+    type: number
+    default: 0
+  - name: PublishCodeCoverage
+    type: boolean
+    default: false
+  - name: Location
+    type: string
+    default: ""
+  - name: Clouds
+    type: string
+    default: 'Public'
+  - name: SupportedClouds
+    type: string
+    default: 'Public,Canary'
+  - name: UnsupportedClouds
+    type: string
+    default: ''
+  - name: PreSteps
+    type: object
+    default: []
+  - name: PostSteps
+    type: object
+    default: []
+  - name: CloudConfig
+    type: object
+    default:
+      Public:
+        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+      Preview:
+        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
+      Canary:
+        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+        Location: 'centraluseuap'
+      UsGov:
+        SubscriptionConfiguration: $(sub-config-gov-test-resources)
+      China:
+        SubscriptionConfiguration: $(sub-config-cn-test-resources)
+  - name: MatrixConfigs
+    type: object
+    default:
+      - Name: Js_live_test_base
+        Path: eng/pipelines/templates/stages/platform-matrix.json
+        Selection: sparse
+        GenerateVMJobs: true
+  - name: AdditionalMatrixConfigs
+    type: object
+    default: []
+  - name: MatrixFilters
+    type: object
+    default:
+      # Include all test scenarios except min/max tests by default
```

Comment:
OK interesting this is how you handled excluding part of the default matrix by default.

Reply:
@benbp I assume you still plan to remove this default based on our last conversation so I'll wait to review the rest of this PR until that change happens as it will impact the other files.

Reply:
Yep, this PR is pending filter changes. I've been in the org bootcamp, so getting back to this today.